pkgs/development/mobile/androidenv/mkrepo.rb

```diff
@@ -281,8 +281,18 @@ def parse_addon_xml doc
   [licenses, addons, extras]
 end
 
+def merge_recursively a, b
+  a.merge!(b) {|key, a_item, b_item|
+    if a_item.class == Hash and b_item.class == Hash
```

Comment:
a_item.is_a?(Hash) && b_item.is_a?(Hash)

Reply:
Fixed it in [3f37ef2](https://github.com/NixOS/nixpkgs/pull/208137/commits/3f37ef2d4696087334b646e3003bcb0833a86a32)